SConstruct

```diff
@@ -1048,6 +1048,7 @@ env['has_demangle'] = conf.CheckDeclaration("boost::core::demangle",
                                 '#include <boost/core/demangle.hpp>', 'C++')
 
 import SCons.Conftest, SCons.SConf
+from distutils.version import LooseVersion
```

Comment:
You don't need to import this, `LooseVersion` is imported somewhere in the `builtutils`, which is `*` imported in `SConstruct`.

Reply:
Removed.